Angular/src/app/paradigm-list/paradigm-list.component.ts

```diff
@@ -38,4 +38,8 @@ export class ParadigmListComponent implements OnInit {
     }
   }
 
+  exportAffixFile() {
+    this.data.getAffix(this.language_name).subscribe();
```

Comment:
Awesome, thanks for adding this. I'll look into how we can handle the returned output a bit as well!